resources/views/pages-legacy/globalRanking.blade.php

```diff
@@ -245,7 +256,7 @@
                 $rank = $rowRank;
             }
 
-            $rankPoints = $dataPoint['Points'];
+            $rankValue = $dataPoint[$rankField];
```

Comment:
This works fine for matches within a page, but doesn't carry over between pages.

o=10000:
![image](https://github.com/user-attachments/assets/88f64732-6f8e-4b68-ac10-cb498458444d)

o=10025:
![image](https://github.com/user-attachments/assets/8bb3dd51-6069-40e9-ab66-dc3760496154)

The switch on line 253 would need to be updated for the additional cases.

I'm not certain it's important enough to address.

Reply:
I've picked away at this one during two different working sessions on the branch, and I'm struggling to come up with a solution that both works and doesn't result in an extremely long query execution time.

I'll probably give it another shot over the weekend, but this is proving to be tough to resolve.

Reply:
If it's too much trouble, it's probably not worth the effort. How often are people going to be looking for an exact rank by number of achievements unlocked where there's a discrepancy caused by pagination?